Classes/Command/TradosCommandController.php

```diff
@@ -4,7 +4,7 @@
 /*
  * This file is part of the Flownative.Neos.Trados package.
  *
- * (c) Flownative GmbH - https://www.flownative.com/
+ * (c) Contributors of the Neos Project - www.neos.io
```

Comment:
Nope…
```suggestion
 * (c) Flownative GmbH - https://www.flownative.com/
```

Reply:
your repository, your rules.. I'll change it back.
I thought it's fair to have your company header when a package is exclusively developed by it, but when it becomes more of a community project, I could change it to the community header.